src/Mealz/MealBundle/Controller/MealController.php

```diff
@@ -325,19 +341,22 @@ private function getParticipantCountMessage(Exception $error): string
     }
 
     /**
+     * Adds a participation for every chosen meal.
```

Comment:
Almost the method name, I would remove it, because there is no benefit from that comment.